pkg/api/getambassador.io/v2/authservice_types.go

```diff
@@ -51,6 +51,7 @@ type AuthServiceSpec struct {
 	AllowedRequestHeaders       []string                  `json:"allowed_request_headers,omitempty"`
 	AllowedAuthorizationHeaders []string                  `json:"allowed_authorization_headers,omitempty"`
 	AddAuthHeaders              map[string]BoolOrString   `json:"add_auth_headers,omitempty"`
+	InitialMetadata             map[string]BoolOrString   `json:"initial_metadata,omitempty"`
```

Comment:
Would it make sense to just make this a `map[string]string`? `AddAuthHeaders` uses the `bool` option in a very specific way, and we generally would rather not add more of those...

Reply:
done